src/state/Send.ts

```diff
@@ -144,14 +146,19 @@ export const send: ISendModel = {
       name,
       multiPathPaymentsEnabled,
     );
+
+    
     log.i("status", [sendPaymentResult.status, sendPaymentResult.failureReason]);
     if (sendPaymentResult.status !== lnrpc.Payment.PaymentStatus.SUCCEEDED) {
       throw new Error(`${translatePaymentFailureReason(sendPaymentResult.failureReason)}`);
     }
+    
+    const settlementDuration = (new Date().getTime() - start);
 
     const transaction: ITransaction = {
       date: paymentRequest.timestamp,
       description: extraData.lnurlPayTextPlain ?? paymentRequest.description,
+      duration: settlementDuration,
```

Comment:
Change to this otherwise web version breaks for unknown reason.

```suggestion
      duration: settlementDuration ?? null,
```